scribbler/tests/test_views.py

```diff
@@ -392,3 +398,42 @@ def test_permission_required(self):
         data = self.get_valid_data()
         response = self.client.post(self.url, data=data)
         self.assertEqual(response.status_code, 403)
+
+
+@override_settings(ROOT_URLCONF='scribbler.tests.urls')
+class FunctionalTestCase(StaticLiveServerTestCase, BaseViewTestCase):
+
+    def setUp(self):
+        super(FunctionalTestCase, self).setUp()
+        self.browser = webdriver.Firefox()
+        self.browser.implicitly_wait(3)
+
+    def tearDown(self):
+        self.browser.quit()
+
+    def test_editor(self):
+        self.browser.get('%s%s' % (self.live_server_url, '/test/'))
+        username_input = self.browser.find_element_by_name("username")
+        username_input.send_keys('test')
+        password_input = self.browser.find_element_by_name("password")
+        password_input.send_keys('test')
+        self.browser.find_element_by_name('submit').click()
+        scribble = self.browser.find_element_by_class_name("scribble-wrapper")
+        self.assertTrue(scribble)
+        editor = self.browser.find_element_by_id("scribbleEditorContainer")
+        self.assertTrue(editor)
+        scribble.click()
+        # The textarea in the editor is inside a div with height=0 which selenium views as not visible
+        # and therefore won't send keys to, so we have to give that div a non-zero height.
+        self.browser.execute_script("return document.getElementsByClassName('CodeMirror cm-s-default')[0].firstChild.style.height='1px';")
+        time.sleep(1)
```

Comment:
It would be nice if we could disable JQuery animations instead of using a time.sleep() here.